test/functional/cmdLineTests/criu/playlist.xml

```diff
@@ -204,12 +192,6 @@
 			<variation>-XX:+DebugOnRestore -Xjit</variation>
 			<variation>-XX:+DebugOnRestore -Xjit:count=0</variation>
 		</variations>
-		<disables>
```

Comment:
There is another `disable` section added later at L168, please remove it as well

Reply:
This is a revert. I do not think I can directly change it. I will reenable L168 in a separate PR.

Reply:
I expect you can (force-)push other changes to the branch behind, but if that other `<disable>` was added via a separate pull request, this should be clear that it reverts multiple changes.

Reply:
updated